docs/reference/github/workflow/pr.md

```diff
@@ -0,0 +1,82 @@
+# Pull Requests
+
+Pull requests are used to facilitate code reviews when merging a branch into the main branch. A pull request
+is the implementation of an issue or other change that we want to make to the main branch.
+
+!!! note
+
+    Pull requests are much like issues where we can do many of the same things. This goes for creating
+    comments in markdown, assigning reviewers, adding labels, adding projects, or adding milestones.
+    Sometimes we skip writing an issue when the change is relatively small.
+
+## Creating a pull request
+
+To create a pull request in a repository, to go the `Pull requests` tab and then click `New pull request`:
+
+![image](../../../assets/images/github/workflow/create_pr.png)
+
+On the next screen, you need to select the base branch that you are merging into, and the branch that you
+are comparing. For the most part, the base branch will be the main branch, and the branch that you are comparing
+will be the issue branch.
+
+![image](../../../assets/images/github/workflow/branch_comparison.png)
+
+Once you have decided on your base and compare branches, click on `Create pull request`. You should see
+the page below:
+
+![image](../../../assets/images/github/workflow/pr_example.png)
+
+Notice how this is remarkably similar to the page of an issue. To link a pull request to an issue, simply add `#<ISSUE NUMBER>`
+to the initial comment in the pull request.
+
+!!! example
+
+    "This issue resolves #49. Please review my pull request!"
+
+Observe that the right-hand side banner contains the following:
+
+| Field               | Description                                                                                        |
+| :------------------ | :------------------------------------------------------------------------------------------------- |
+| Reviewers           | Assign reviewers to review your pull request. Always try to assign at least one reviewer.          |
+| Assignees           | Assign the people who worked on the issue.                                                         |
+| Labels              | Assign labels to categorize pull requests.                                                         |
+| Projects            | Assign a pull request to a project.                                                                |
+| Milestone           | Assign a pull request to a milestone.                                                              |
```

Comment:
One more thing, you could link to where you talk about a field in the issues sectio; for example `[Labels](./issues.md#labelling-issues)`